cmake/std/trilinosprhelpers/setenvironment/unittests/setenvironment_config.ini

```diff
@@ -152,3 +154,50 @@ module-unload gcc:
 
 [Module_Purge]:
 module-purge:
+
+
+#
+# Sections to test cycle breaking
+# - Test by trying to process Sec_D, we should get blocked
+#   when Sec_A tries to load Sec_C
+# - Final result should have envvar Sec_C == '99'
+[Sec_A]:
+setenv Sec_A: 1
+setenv Sec_C: 99
+use Sec_C:
```

Comment:
Do you want the environment variable names to be the same as the section names?  Might be a source of confusion in the future.

Reply:
it doesnt matter